packages/create-app/templates/default-app/packages/backend/src/plugins/search.ts

```diff
@@ -49,6 +50,9 @@ export default async function createPlugin({
 
   return await createRouter({
     engine: indexBuilder.getSearchEngine(),
+    types: indexBuilder.getDocumentTypes(),
```

Comment:
This change needs a changeset with the usual instructions for how to upgrade an existing app

Reply:
:+1: done in 9adac61b42.